R/pkg/R/sparkR.R

```diff
@@ -123,16 +123,24 @@ sparkR.init <- function(
     uriSep <- "////"
   }
 
+  sparkEnvirMap <- convertNamedListToEnv(sparkEnvir)
+
   existingPort <- Sys.getenv("EXISTING_SPARKR_BACKEND_PORT", "")
   if (existingPort != "") {
     backendPort <- existingPort
   } else {
     path <- tempfile(pattern = "backend_port")
+    # A few Spark config cannot be set in env:
```

Comment:
wording:
A few Spark properties can not take effect after the driver JVM has started, as documented in 
http://spark.apache.org/docs/latest/configuration.html#application-properties

In SparkR shell mode, add them to spark-submit commandline if not already set in SPARKR_SUBMIT_ARGS so that they can take effect when launching the JVM.